samples/csharp_dotnetcore/14.nlp-with-dispatch/wwwroot/default.html

```diff
@@ -0,0 +1,10 @@
+﻿<!DOCTYPE html>
```

Comment:
Should't we use the same html file here as the other samples?

Reply:
probably ...second pass - we are focusing on structure for this pass